glide.yaml

```diff
@@ -1,3 +1,8 @@
 package: github.com/wantedly/developers-account-mapper
 import:
 - package: github.com/mitchellh/cli
+- package: github.com/aws/aws-sdk-go
+  subpackages:
+  - aws
+  - aws/session
+  - aws/service/dynamodb
```

Comment:
It's not wrong. Maybe slack-mention-converter one is that we have forgotten doing `glide update`. No matter how many lines added, it's ok. Please see https://github.com/dtan4/s3url/blob/master/glide.lock .

Actually, we just want to keep __revision__ of dependencies.